tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -33,6 +33,12 @@ const LOCAL_HOSTNAMES: string[] = ['localhost', '127.0.0.1'];
   `,
   styles: [
     `
+      /* More precise selector than angular material selectors to override
+         styling. */
+      button.mat-stroked-button {
+        background-color: rgba(198, 202, 209, 0.2);
+        border: 1px solid #e9e9e9;
```

Comment:
Please use SASS and the variables from the theme. Also for the rgba, use the rgba mixin and specify using the hex (prefer theme variable, too).

Reply:
I refactored CSS and HTML to their own files.

There was much offline discussion and eventually we landed on making the button color Material Orange 500.  I've made some modification to the $tf-orange palette and $tb-primary to incorporate the new color. Those changes may require more discussion. 

Now I can set `background-color: mat-color($tb-primary, 500);` and the colors work with the tf-orange palette and sort of work with the tf-slate palette.